### DISCUSSION TITLE
DCDAO MEDIUM GROWTH PLAN

### DISCUSSION THREAD

**Discussion by JuneCribb** [ringdao-community-guild/discussion#104] [General]
**Overview**

During my trial stage as an advocate, I proposed to Cindy about growing the DCDAO medium page. 
And that was my task, to write articles for DCDAO and I don't think that was enough to grow the page. 
I've reached out to our community manager and I'm proposing generally for the community to help in the growth of DCDAO medium page. 

**How do I plan to do that?**

With the aid of the community and team, we can grow the page by; 

- Update campaigns going on in DCDAO.
- Write Monthly Report. 
- Include the page for members to follow/ subscribe to during campaigns. (important)
- Additionally, educate the community about Darwinia tech/ DCDAO and general crypto. 

DCDAO is growing including its X account, adding Medium to its growth will be good. 

Let me know what you think about this.

**Comment by SasoLithops** [comment]
Absolutely for this. Medium articles are for me one of best ways to research project. And I am ALWAYS for education.
